src/external-server/external-server.ts

```diff
@@ -177,8 +179,10 @@ export abstract class ExternalServer {
     }
 
     public async post<T>(data: StringMap): Promise<T> {
+        await this.ensurePort();
         await this.ensureStarted();
 
+        axiosRetry(axios, { retries: 5, retryDelay: axiosRetry.exponentialDelay });
```

Comment:
Add a comment here about it not working in CircleCI with `retries: 3`. Can you find out how much time these 5 retries actually amount to?

Reply:
https://github.com/0xSpaceShard/starknet-hardhat-plugin/pull/374/commits/e7b25f93f7cf0ee5428958cf1b3e7469586a624d